.eslintrc

```diff
@@ -23,6 +23,7 @@
   }],
   "rules": {
     "oppia/break-after-parens": "error",
+    "oppia/test-message-style": "error",
```

Comment:
I'm surprised to see that there isn't any existing lint issue in the codebase for this check! Can you please do an audit and confirm that it's true?

Reply:
Just as an FYI, I tried running `git grep 'it(.[^s]'` and I see this log:
![image](https://user-images.githubusercontent.com/16653571/91892586-8cde4e80-ecb0-11ea-88d6-f718d659037c.png)

There are many more in the codebase, can you please check why this new check didn't catch any error!

Reply:
Yes, there aren't any existing lint issues in the codebase, I've checked it by making an error intentionally and the lint check failed there.

Reply:
haha... we added opposite comment around the same time! Can you please check my 2nd comment in this thread?

Reply:
Working on it

Reply:
@DubeySandeep The check is working fine, If I run lint check on a single file in which a lint error exists, it is catching that error successfully. But while running on whole codebase it is not catching errors. Strange! seems like an error in eslint

Reply:
> seems like an error in eslint

Is it an issue with any other checks? If not, then it's more specific to the new check we have written!

Reply:
I don't think the problem is with a check because I tried changing some files and running linter on them. The check is reporting all errors successfully. But when the linter run over all files it is not catching errors. I've also seen this with no-unused-vars and import-order where I had to run linter almost 30 or 40 times

Reply:
FYI, ESLint is throwing error on CI:
![image](https://user-images.githubusercontent.com/16653571/91896984-49d3a980-ecb7-11ea-8c67-b8ccefb0d627.png)


Can you please check what's going on and why it didn't result in a failure of the lint test?

Reply:
I will investing what might be causing this error. Thanks for the info

Reply:
Fixed the error

Reply:
I still don't see any changes in existing test messages which are not correct! Am I missing something here?

Reply:
The error are being reported now, I will fix the errors tomorrow

Reply:
@Hudda, I'll take a quick look but I would suggest you fix the issue asap so that I can do a full review!

Reply:
Done.

Reply:
> Can you please check what's going on and why it didn't result in a failure of the lint test?

This thread is still not resolved! Can you please reply to this comment^?

Reply:
We are using sys.exit(1) to exit if there is an error with the linter. We need to return the TaskResult object as well to show the error. But I think this can be done in a separate PR. What do you think?

Reply:
This doesn't explain why the lint test passed when there is some error in lint test execution! I was expecting the lint test to fail if we do sys.exit(1).

Reply:
because js_ts_linter is a sublinter to the main linter even if we exit from a sublinter the other sublinters continue to work and the linter pass.

Reply:
hmmm... I don't think this is a case for other linter. This way our lint test will never fail if there's an error in any of the linter, right? I think we have a structure to fail the test if some linter error happens, can you please check? (Create a separate issue and assign it to yourself and let's fix this asap!)

Reply:
Created the issue https://github.com/oppia/oppia/issues/10633